redis/src/main/scala/zio/redis/api/Keys.scala

```diff
@@ -346,6 +393,37 @@ private[redis] object Keys {
       ScanOutput
     )
 
+  final val Sort
+    : RedisCommand[(String, Option[String], Option[Limit], List[String], Order, Option[Alpha]), Chunk[String]] =
+    RedisCommand(
+      "SORT",
+      Tuple6(
+        StringInput,
+        OptionalInput(ByInput),
+        OptionalInput(LimitInput),
+        ListInput(GetInput),
```

Comment:
This can actually be changed to `OptionalInput(NonEmptyList[GetInput])`.

Reply:
clever!